HPXMLtoOpenStudio/resources/battery.rb

```diff
@@ -40,8 +40,15 @@ def self.apply(model, battery)
     battery_surface_area = 0.306 * (nominal_capacity_kwh**(2.0 / 3.0)) # m^2
 
     # Assuming 3/4 of unusable charge is minimum SOC and 1/4 of unusable charge is maximum SOC, based on SAM defaults
-    minimum_storage_state_of_charge_fraction = 0.75 * (1.0 - usable_fraction)
-    maximum_storage_state_of_charge_fraction = 1.0 - 0.25 * (1.0 - usable_fraction)
+    unusable_fraction = 1.0 - usable_fraction
+    minimum_storage_state_of_charge_fraction = 0.75 * unusable_fraction
+    maximum_storage_state_of_charge_fraction = 1.0 - 0.25 * unusable_fraction
+
+    # disable voltage dependency unless lifetime model is requested: this prevents some scenarios where changes to SoC didn't seem to reflect charge rate due to voltage dependency and constant current
+    voltage_dependence = false
+    if battery.lifetime_model == HPXML::BatteryLifetimeModelKandlerSmith
+      voltage_dependence = true
+    end
```

Comment:
@jmaguire1 Are we sure that we want to continue to allow the lifetime model? I am concerned that it will provide unexpected results, and I question how meaningful it is to have a model that reflects only a single year of the battery's multi-year operation. Unless you can convince me otherwise, I would recommend that we remove this input.

Reply:
Moved to a separate issue (https://github.com/NREL/OpenStudio-HPXML/issues/1211), we don't have to hold up this PR for it.